docs/main/Solutioning/openebs-on-kubernetes-platforms/rwx.md

```diff
@@ -0,0 +1,502 @@
+
+## Details of Setup
+
+### Setting up a Single NFS Server
+
+1. Create a Replicated PV Mayastor Pool.
+
+  Create a Replicated PV Mayastor pool that satisfies the performance and the availability requirements. See [Replicated PV Mayastor Configuration documentation](../../user-guides/replicated-storage-user-guide/replicated-pv-mayastor/rs-configuration.md#create-diskpools) for more details.
+
+  **Example of a Replicated PV Mayastor Pool**
+
+  **Command**
+
+  ```
+  $ kubectl get dsp -n mayastor
+  ```
+
+  **Output**
+
+  ``` 
+  NAME     NODE                                       STATE     POOL_STATUS   CAPACITY      USED   AVAILABLE
+  pool-1   gke-gke-ssd-1-default-pool-d802bc13-3j4s   Created   Online        10724835328   0      10724835328
+  pool-2   gke-gke-ssd-1-default-pool-d802bc13-z9hf   Created   Online        10724835328   0      10724835328
+  pool-3   gke-gke-ssd-1-default-pool-d802bc13-67bv   Created   Online        10724835328   0      10724835328
+  ```
+
+2. Create a Replicated PV Mayastor Storage Class.
+
+  Create a storage class to point to the above created pool. Also, select the number of replicas and the default size of the volume. See [Replicated PV Mayastor Configuration documentation](../../user-guides/replicated-storage-user-guide/replicated-pv-mayastor/rs-configuration.md#create-replicated-pv-mayastor-storageclasss) for more details.
+
+  **Example of a Replicated PV Mayastor Storage Class**
+
+  **Command**
+
+  ```
+  $ kubectl get sc
+  ```
+
+  **Output**
+
+  ```
+  mayastor-2                io.openebs.csi-mayastor   Delete          Immediate              false                  24h
+  mayastor-3                io.openebs.csi-mayastor   Delete          Immediate              false                  27h
+  mayastor-etcd-localpv     openebs.io/local          Delete          WaitForFirstConsumer   false                  27h
+  mayastor-loki-localpv     openebs.io/local          Delete          WaitForFirstConsumer   false                  27h
+  mayastor-single-replica   io.openebs.csi-mayastor   Delete          Immediate              true                   27h
+  ```
+
+3. Create a namespace for deploying the NFS server.
+
+  **Command**
+  
+  ```
+  $ kubectl create ns nfs-server
+  ```
+
+  **Output**
+  
+  ```
+  $ kubectl create ns nfs-server
+  namespace/nfs-server created
+  ```
+
+4. Create a PVC with a desired storage for the NFS server using the Replicated PV Mayastor storage class.
+
+  **Command**
+
+  ```
+  $ cat <<EOF | kubectl create -f -
+  ```
+
+  **Output**
+
+  ```
+  apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    name: nfs-server-claim
+    namespace: nfs-server
+  spec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 5Gi
+    storageClassName: mayastor-3
+  EOF
+  persistentvolumeclaim/nfs-server-claim created
```

Comment:
Only this line is output rest all are command itself. refer confluence for the same.